cmd/snap-confine/udev-support.c

```diff
@@ -443,9 +430,39 @@ void sc_setup_device_cgroup(const char *security_tag)
 		return;
 	}
 	/* Setup the device group access control list */
-	sc_udev_setup_acls(fds.devices_allow_fd, fds.devices_deny_fd,
-			   udev, assigned);
+	sc_udev_setup_acls_common(fds.devices_allow_fd, fds.devices_deny_fd,
+				  udev, assigned);
+	for (struct udev_list_entry * entry = assigned; entry != NULL;
+	     entry = udev_list_entry_get_next(entry)) {
+		const char *path = udev_list_entry_get_name(entry);
+		if (path == NULL) {
+			die("udev_list_entry_get_name failed");
+		}
+		struct udev_device *device =
+		    udev_device_new_from_syspath(udev, path);
+		/** This is a non-fatal error as devices can disappear asynchronously
+		 * and on slow devices we may indeed observe a device that no longer
+		 * exists.
+		 *
+		 * Similar debug + continue pattern repeats in all the udev calls in
```

Comment:
Thanks for the comment! I was exactly wondering if all those `debug()` calls in `sc_udev_allow_assigned_device()` should be something more serious, but this clarifies it.